LICENSE

```diff
@@ -1,3 +1,13 @@
+The website code, configuration, and supporting technical files are licensed
```

Comment:
```suggestion
The content of this repository is licensed
```

Keep it as unrestrictive as possible. We don't want to only cover part of the stuff via this license but everything in this repository.